sentry/src/main/java/io/sentry/config/PropertiesProvider.java

```diff
@@ -74,4 +74,16 @@ default Double getDoubleProperty(final @NotNull String property) {
     final String result = getProperty(property);
     return result != null ? Double.valueOf(result) : null;
   }
+
+  /**
+   * Resolves a {@link Long} property given by it's name.
+   *
+   * @param property - the property name
+   * @return property value or the default value if not found.
+   */
+  @Nullable
+  default Long getLongProperty(final @NotNull String property) {
+    final String result = getProperty(property);
+    return result != null ? Long.valueOf(result) : null;
```

Comment:
That can throw `NumberFormatException`, Maybe we wrap in a try catch?

Reply:
If possible yes, log out if there's a logger instance available and return null.